src/applications/disability-benefits/686/config/form.js

```diff
@@ -122,6 +122,90 @@ const addressSchema = {
   },
 };
 
+const dependentTypeSchema = {
+  type: 'string',
+  enum: ['SPOUSE', 'DEPENDENT_PARENT', 'CHILD'],
+  enumNames: ['Spouse', 'Dependent Parent', 'Child'],
+};
+
+const dependentTypeSchemaUI = {
+  'ui:title': "What was your dependent's status?",
+  'ui:widget': 'radio',
+};
+
+const childStatusSchema = {
+  type: 'object',
+  properties: {
+    childUnder18: {
+      type: 'boolean',
+    },
+    stepChild: {
+      type: 'boolean',
+    },
+    adopted: {
+      type: 'boolean',
+    },
+    disabled: {
+      type: 'boolean',
+    },
+    childOver18InSchool: {
+      type: 'boolean',
+    },
+  },
+};
+
+const childStatusUiSchema = {
+  'ui:title': "Child's status (Check all that apply)",
+  'ui:required': (formData, index) =>
+    formData.deaths[`${index}`].dependentType === 'CHILD',
+  'ui:options': {
+    expandUnder: 'dependentType',
+    expandUnderCondition: 'CHILD',
+    showFieldLabel: true,
+    keepInPageOnReview: true,
+  },
+  childUnder18: {
+    'ui:title': 'Child under 18',
+  },
+  stepChild: {
+    'ui:title': 'Stepchild',
+  },
+  adopted: {
+    'ui:title': 'Adopted child',
+  },
+  disabled: {
+    'ui:title': 'Child incapable of self-support',
+  },
+  childOver18InSchool: {
+    'ui:title': 'Child 18-23 and in school',
+  },
+};
+
+const deathLocationSchema = {
+  type: 'object',
+  properties: {
+    city: {
+      type: 'string',
+      maxLength: 30,
+      pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$',
+    },
+    state: {
+      type: 'string',
+      maxLength: 30,
+      pattern: '^(?!\\s)(?!.*?\\s{2,})[^<>%$#@!^&*0-9]+$',
+    },
+  },
+};
```

Comment:
This, and all of its references, is changing to `locationSchema` pending [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/11534)